guides/v2.1/cloud/env/variables-intro.md

```diff
@@ -64,7 +66,9 @@ return array(
    ...
 );
 ```
-{% include note.html type="info" content="JS bundling and JS/CSS merging do not work with SCD on demand." %}
+
+{: .bs-callout .bs-callout-info}
+JS bundling and JS/CSS merging do not work with SCD on demand.
```

Comment:
we might have to double check this warning

Reply:
Yeah, I don't see it in the DIST packaged in ece-tools. Will remove.